guacamole_ec2.tf

```diff
@@ -52,7 +53,7 @@ resource "aws_instance" "guacamole" {
     # Restrict put responses from IMDS to a single hop (this is the
     # default value).  This effectively disallows the retrieval of an
     # IMDSv2 token via this machine from anywhere else.
-    http_put_response_hop_limit = 1
+    http_put_response_hop_limit = 2
```

Comment:
The comment above should be updated to explain why the hop limit has been bumped up to 2.

Reply:
Please see commit 0969c94.

Reply:
Great comment! 📜

Reply:
Not for nothing... but does it make sense to run the container with `host` networking instead of `bridge` mode.  This would allow us to leave the IDMS hop limit at `1`.  

https://docs.docker.com/network/host/

I'm not asking to change this now, but it might be a discussion worthy of our collective time.

Reply:
How does that work with respect to one container being able to see the other containers in the composition?